zkevm-circuits/src/evm_circuit/util/common_gadget/curie.rs

```diff
@@ -41,18 +41,18 @@ impl<F: Field> CurieGadget<F> {
         );
 
         // TODO: refactor
-        // is_scoll_chain means (chain_id - 534352) * (chain_id - 222222) == 0
-        let is_scroll_chain = IsZeroGadget::construct(
+        // is_morph_chain means (chain_id - 53077) * (chain_id - 2710) == 0
```

Comment:
Wrong MAINNET_CHAIN_ID in comment, -> is_morph_chain means (chain_id - 2818)*(chain_id - 53077)== 0

Reply:
it should be `(chain_id - 2818)*(chain_id - 53077)*(chain_id-2710) == 0`

Reply:
But you only check the chain_id equals to  MORPH_MAINNET_CHAIN_ID or MORPH_DEVNET_CHAIN_ID, without checking the TESTNET CHAIN ID.